containers/Ideas/IdeaCreate.js

```diff
@@ -104,13 +105,29 @@ export const IdeaCreate = props => {
 
   const handleSubmit = (values) => {
     values.labels = getSelectedLabels();
+    let formData = new FormData();
+    /* FormData requires name: id */
+    // formData.append("website", "question");
+
+    /* append input field values to formData */
+    for (let value in values) {
+      formData.append(value, values[value]);
+    }
+    formData.append('image', { uri: image.uri, name: image.name, type: 'image/jpeg' });
```

Comment:
I think the type also needs to become dynamic. We might need to use a library for that, though.